examples/metrics/metrics/observer.js

```diff
@@ -14,29 +15,75 @@ const exporter = new PrometheusExporter(
 
 const meter = new MeterProvider({
   exporter,
-  interval: 2000,
+  interval: 3000,
```

Comment:
Just curious to know, why we have to increase the interval?

Reply:
it was just testing, debugging etc., not important